packages/front-end/pages/metric/[mid].tsx

```diff
@@ -161,21 +274,62 @@ const MetricPage: FC = () => {
       </div>
 
       <div className="row align-items-center mb-2">
-        <h1 className="col-auto">{metric.name}</h1>
+        <h1 className="col-auto">
+          {metric.name}{" "}
+          {metric.status === "archived" && (
+            <small className="text-muted">(archived)</small>
+          )}
+        </h1>
         <div style={{ flex: 1 }} />
         {canEdit && (
           <div className="col-auto">
-            <DeleteButton
-              className="ml-2"
-              onClick={async () => {
-                await apiCall(`/metric/${metric.id}`, {
-                  method: "DELETE",
-                });
-                mutateDefinitions({});
-                router.push("/metrics");
-              }}
-              displayName={"Metric '" + metric.name + "'"}
-            />
+            {metric.status === "archived" ? (
+              <DeleteButton
+                className="ml-2"
+                text="Delete/unarchive"
+                title="Delete/unarchive this metric"
+                getConfirmationContent={getMetricUsage(metric)}
+                secondaryAction="un-archive"
+                secondaryActionClick={async () => {
+                  await apiCall(`/metric/${metric.id}/status?status=active`, {
+                    method: "PUT",
+                  });
+                  mutateDefinitions({});
+                  router.push("/metrics");
+                }}
```

Comment:
This feels weird to have the "un-archive" option live within the delete modal.  I would be scared to click the delete button.

Why don't we switch the metric page to have a 3-dot dropdown menu like experiments.  Then we can put separate CTAs for delete and archive/unarchive.